modules/service/src/main/scala/lucuma/odb/service/TargetService.scala

```diff
@@ -172,7 +171,7 @@ object TargetService {
         val stmt = Statements.cloneTarget(pid, targetId, user)
         session.prepareR(stmt.fragment.query(target_id)).use(_.option(stmt.argument))
 
-      def cloneTargetImpl(input: CloneTargetInput)(using Transaction[F]): F[CloneTargetResponse] =
+      private def cloneTargetImpl(input: CloneTargetInput)(using Transaction[F]): F[CloneTargetResponse] =
```

Comment:
Scala 2 knows that non-interface methods on anonymous classes are implicitly private and warns if they're unused. Scala 3 isn't that smart and you can end up with unreachable code unless you mark everything private.

Reply:
I talked to Som Snytt about this and he's looking into it.